src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
The interface name `AreaCalculator` is a verb phrase, which is not ideal for interface names. Consider using a noun or a noun phrase that represents the capability, such as `AreaComputable` or `AreaProvider`.